packages/web/src/client/internal/open-feature-client.ts

```diff
@@ -317,4 +333,13 @@ export class OpenFeatureClient implements Client {
   private get _logger() {
     return this._clientLogger || this.globalLogger();
   }
+
+  private shortCircuitIfNotReady() {
```

Comment:
Perhaps we could move this to a utils method in core to avoid duplication.

Reply:
See: https://github.com/open-feature/js-sdk/pull/1020#discussion_r1804558887